Resources/Locale/en-US/deltav/navmap-beacons/station-beacons.ftl

```diff
@@ -0,0 +1,28 @@
+station-beacon-epistemics = Epistemics
+
+station-beacon-logistics = Logistics
+station-beacon-logistics-reception = Logistics Reception
+station-beacon-lo = LO's Room
```

Comment:
Again redundant since there are already beacons for this

Reply:
I assumed you just did it this way to get through the big merge a little quicker. Would this not fit our file standards better and allow for keeping track of upstream file changes?